man/systemd.special.xml

```diff
@@ -130,9 +130,20 @@
           for this target unit to all services (except for those with
           <varname>DefaultDependencies=no</varname>).</para>
 
-          <para>Usually this should pull-in all mount points, swap
-          devices, sockets, timers, and path units and other basic
-          initialization necessary for general purpose daemons.</para>
+          <para>Usually this should pull-in all local mount points plus
+          <filename>/var</filename>, <filename>/tmp</filename> and
```

Comment:
That's redundant.  If they are mount points, they are covered by "all mount points" and if not, there is nothing to pull in.